app/platform/service_test.go

```diff
@@ -124,7 +124,6 @@ func TestMetrics(t *testing.T) {
 	})
 
 	t.Run("ensure the metrics server is started with advanced metrics", func(t *testing.T) {
-		t.Skip("MM-47635")
```

Comment:
This seems unrelated?

Reply:
It was part of one of the PRs I cherry-picked the commit to fix the issue.